src/posit/connect/system.py

```diff
@@ -0,0 +1,291 @@
+        """
+        List all content runtime caches.
+
+        List all content runtime caches. These include packrat and Python
+        environment caches.
+
+        This information is available only to administrators.
+
+        Returns
+        -------
+        List[SystemRuntimeCache]
+            List of all content runtime caches.
+
+        Examples
+        --------
+        ```python
+        from posit.connect import Client
+
+        client = Client()
+
+        runtime_caches = client.system.caches.runtime.find()
+        ```
+        """
+        url = self._ctx.url + self._path
+        response = self._ctx.session.get(url)
+        results = response.json()
+        if not isinstance(results, dict) and "caches" not in results:
+            raise RuntimeError("`caches=` not found in response.")
+        caches = results["caches"]
+        return [SystemRuntimeCache(self._ctx, self._path, **cache) for cache in caches]
+
+    @overload
+    def destroy(
+        self, cache: SystemRuntimeCache, /, *, dry_run: Literal[False] = False
+    ) -> Task: ...
+    @overload
+    def destroy(self, cache: SystemRuntimeCache, /, *, dry_run: Literal[True]) -> None: ...
+    @overload
+    def destroy(
+        self,
+        /,
+        *,
+        language: str,
+        version: str,
+        image_name: str,
+        dry_run: Literal[False] = False,
+    ) -> Task: ...
+    @overload
+    def destroy(
+        self,
+        /,
+        *,
+        language: str,
+        version: str,
+        image_name: str,
+        dry_run: Literal[True] = True,
+    ) -> None: ...
+
+    def destroy(
+        self,
+        cache: Optional[SystemRuntimeCache] = None,
+        /,
+        **kwargs,
+    ) -> Task | None:
+        """
+        Delete a content runtime package cache.
+
+        Delete a content runtime package cache by specifying language, version, and execution
+        environment.
+
+        This action is only available to administrators.
+
+        Parameters
+        ----------
+        cache : SystemRuntimeCache
+            The system runtime cache object to destroy.
+        language : str
+            The runtime language of the cache.
+        version : str
+            The language version of the cache.
+        image_name : str
+            The name of the cache's execution environment.
+        dry_run : bool, optional
+            If `True`, the cache will not be destroyed, only the operation will be simulated.
+
+        Returns
+        -------
+        Task | None
+            The task object if the operation was successful. If `dry_run=True`, `None` is returned.
+
+        Examples
+        --------
+        ```python
+        from posit.connect import Client
+
+        client = Client()
+
+        runtime_caches = client.system.caches.runtime.find()
+        first_runtime_cache = runtime_caches[0]
+
+        # Remove the cache
+        task = client.system.caches.runtime.destroy(first_runtime_cache, dry_run=False)
```

Comment:
Should the user just run `first_runtime_cache.destroy()` instead? Not sure if this method is adding additional benefit to the API.

Reply:
Agreed. Removing!